HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -260,14 +279,14 @@ private String writeFile() {
 	private class HVConnect extends AsyncTask<Void, Void, Void> {
 		
 		private ProgressDialog progressDialog;
-        
-        public HVConnect() {
-            progressDialog = ProgressDialog.show(
-                    MainActivity.this,
-                    "",
-                    "Please wait...", 
-                    true);
-        }
+
+		public HVConnect() {
+			progressDialog = ProgressDialog.show(
```

Comment:
nit: the dialog and toast calls don't always need to have each parameter on a new line, most of these calls will fit on one line.